File: backend/MyWebApi/Domain/Models/Article.cs
namespace MyWebApi.Domain.Models
{
    public class Article
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string ImageURL { get; set; }
        public string ImageCredit { get; set; }
        public string Date { get; set; }
        public List<Tag> Tags { get; set; }
        public string Body { get; set; }
    }


}

File: frontend/types/article.ts
import { Tag } from './tag';

export type Article = {
  id: number;
  title: string;
  description?: string; // Optisonal if not all articles have it
  imageURL?: string; // Optional if not all articles have it
  tags: Tag[]; // Array of tags
  body?: string; // Optional for articles that don't include the full body
};
